core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -119,16 +119,33 @@ public LogicalPlan analyze(UnresolvedPlan unresolved, AnalysisContext context) {
   public LogicalPlan visitRelation(Relation node, AnalysisContext context) {
     context.push();
     TypeEnvironment curEnv = context.peek();
-    Table table = storageEngine.getTable(node.getTableName());
+    String catalogName = getCatalogName(node);
```

Comment:
How about use catalogService to resolve the catalog name?. e.g. catalogService.resolve(string tableName)

Reply:
We could do that...but if someone want to escape catalog name with back ticks. That information is available only in ASTBuilder. So still gettingCatalog and TableName from Relation.java.
source = `prometheus.http_requests`
source = promtheus.http_requests.